src/converters/camera.cpp

```diff
@@ -99,6 +99,7 @@ const sensor_msgs::CameraInfo& getCameraInfo( int camera_source, int resolution
     if ( resolution == AL::kVGA )
     {
       static const sensor_msgs::CameraInfo cam_info_msg = createCameraInfoDEPTHVGA();
+      ROS_WARN("VGA resolution is not supported for the depth camera, please use QVGA");
```

Comment:
As this is logging it should include the the ros header with the logging macros.
Should we rephrase to say QVGA or lower ? (is QQVGA a valid resolution for the depth camera on pepper ?)

Reply:
True that there are not much ROS_WARN in Naoqi Driver, it seems that they have preferred to use std::cout. 

What kind of macros do you mean? #include <ros/console.h> ?

Agree for "QVGA or lower"

Reply:
> What kind of macros do you mean? #include <ros/console.h> ?

yep :+1:

> Agree for "QVGA or lower"
 
:+1: